.github/workflows/release.yml

```diff
@@ -361,3 +390,7 @@ jobs:
           tag: ${{ github.ref }}
           file_glob: true  # If set to true, the file argument can be a glob pattern
           file: release/*
+
+      - name: Upload release files on Arduino downloads servers
+        run: aws s3 sync release/ s3://${{ secrets.DOWNLOADS_BUCKET }}${{ env.PLUGIN_TARGET }} --include "*"
```

Comment:
The files are uploaded to `https://downloads.arduino.cc/tmp/CreateBridgeStable/<filename>`